OJPP.js

```diff
@@ -338,11 +338,33 @@ function izrisiRelacijskeGumbe(gumbi) {
             brisiMarkerje();
             menuClose();
         }
+
+
+        // For touch devices - delte button after 2 seconds
+        btn.ontouchstart = () => {
```

Comment:
element.ontouchstart is not good because it does not work on touch-screen desktop computers. See https://github.com/scribble-rs/scribble.rs/pull/250#issuecomment-798286483

Instead, use event listeners, something like `drawingBoard.addEventListener('touchstart', onTouchStart);`